index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Webpage</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="navbar">
+        <ul>
+            <li>About</li>
+            <li>Work</li>
+            <li>Content</li>
+        </ul>
+     </div>
+     <div class="middle">
+        <h1>Hey I am Sam</h1>
+        <p>a web developer</p>
+     </div>
+     <div class="project-section">
+        <h1>These are some of my projects</h1>
```

Comment:
The best practice is to use only one h1 tag on a web page because having multiple h1 tags on a webpage can confuse the search engine and users, as the h1 tag is meant to indicate primary topics.